include/gazebo_motor_model.h

```diff
@@ -111,6 +116,11 @@ class GazeboMotorModel : public MotorModel, public ModelPlugin {
   int motor_number_;
   int turning_direction_;
 
+  int motor_Failure_Number_; /*!< motor_Failure_Number is (motor_number_ + 1) as (0) is considered no_fail. Publish accordingly */
```

Comment:
This already exists on the `gazebo_motor_failure_plugin.h` header. So remove this one and push the comment doc to the `gazebo_motor_failure_plugin.h` header file.

Reply:
kept it here since if someone doesn't use the `gazebo_motor_failure_plugin` and does it without ROS and directly on gazebo topic using the `motor_model`. You still want me to move it to the other plugin?